code/__DEFINES/traits/declarations.dm

```diff
@@ -1154,4 +1154,6 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 /// This item cannot be selected for or used by a theft objective (Spies, Traitors, etc.)
 #define TRAIT_ITEM_OBJECTIVE_BLOCKED "item_objective_blocked"
 
+#define TRAIT_EASY_ATTACH "peg"
```

Comment:
1. This should have a doc comment above it (`/// this trait does this`)
2. "peg" isn't actually indicative of what this trait does, you should rename it (though it's not user facing)

Reply:
oh yeah i found/replaced it from before, it was called "TRAIT_PEG"

Reply:
septic fixed this, resolving.